bot.user.js

```diff
@@ -922,11 +922,151 @@ var bot = window.bot = (function (window) {
             return false;
         },
 
+        populatePts: function () {
+            let x = window.snake.xx + window.snake.fx;
+            let y = window.snake.yy + window.snake.fy;
```

Comment:
Have you been able to figure out what fx/fy are?  I've used both xx, and xx+fx in testing (because the game code itself seems to do +fx +fy) but I really couldn't figure out what they are.